odoo/tools/misc.py

```diff
@@ -1380,6 +1380,26 @@ def _format_time_ago(env, time_delta, lang_code=False, add_direction=True):
     return babel.dates.format_timedelta(-time_delta, add_direction=add_direction, locale=locale)
 
 
+def humanize_number(number, decimal=1):
```

Comment:
I still think humanize if a terrible name.

Reply:
@tde-banana-odoo Maybe `format_number` ? (to be coherent with the similar functions)